fog/overseer/server/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+#![feature(proc_macro_hygiene, decl_macro)]
+
```

Comment:
nit: This crate should have `#[deny(missing_docs)]`, ideally all our crates would have that

Reply:
Done.